packages/governance/test/swingsetTests/contractGovernor/test-governor.js

```diff
@@ -98,11 +98,49 @@ const expectedcontractGovernorStartLog = [
   'updated to "[299792458n]"',
   'MalleableNumber was changed to "[602214090000000000000000n]"',
   'current value of MalleableNumber is 299792458',
+  'Electorate was changed to {"brand":"[Alleged: Zoe Invitation brand]","value":[{"description":"questionPoser","expiry":"[undefined]","fee":"[undefined]","handle":"[Alleged: InvitationHandle]","installation":"[Alleged: Installation]","instance":"[Alleged: InstanceHandle]","zoeTimeAuthority":"[undefined]"}]}',
   'MalleableNumber was changed to "[299792458n]"',
   'Voter Alice validated all the things',
 ];
 
-test('zoe - contract governance', async t => {
+test.serial('contract governance', async t => {
   const dump = await main(t, ['contractGovernorStart']);
   t.deepEqual(dump.log, expectedcontractGovernorStartLog);
 });
+
+const expectedChangeElectorateLog = [
+  '=> voter and electorate vats are set up',
+  '@@ schedule task for:2, currently: 0 @@',
+  'Voter Alice voted for {"noChange":{"key":"main","parameterName":"Electorate"}}',
+  'Voter Bob voted for {"changeParam":{"key":"main","parameterName":"Electorate"},"proposedValue":"[Alleged: Zoe Invitation payment]"}',
+  'Voter Carol voted for {"changeParam":{"key":"main","parameterName":"Electorate"},"proposedValue":"[Alleged: Zoe Invitation payment]"}',
+  'Voter Dave voted for {"changeParam":{"key":"main","parameterName":"Electorate"},"proposedValue":"[Alleged: Zoe Invitation payment]"}',
```

Comment:
so the invitation is available in the question details? Getting details of outstanding questions is not closely held, is it? Is it OK for this invitation to be shared widely?

Reply:
No, that was definitely an oversight. I've fixed it.